utility_analysis_new/metrics.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2022 OpenMined.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Dataclasses with Utility Analysis result metrics."""
+
+import pipeline_dp
+from dataclasses import dataclass
+from typing import List, Optional
+import math
+
+
+@dataclass
+class CountMetrics:
```

Comment:
nit: I would have a very slight preference for calling this "CountUtilityMetrics" since there are so many metrics that we typically discuss (including count being a metric itself). But this is a very optional suggestion.

Reply:
Previously this class has a name: `CountUtilityAnalysisMetrics`, a few days ago in discussion with Mirac, we've decided to drop `UtilityAnalysis`, since anyway those metrics are inside `utility_analysis` package and it doesn't make sense to duplicate utility.